src/views/List.jsx

```diff
@@ -7,6 +8,10 @@ export function List({ data }) {
 				Hello from the <code>/list</code> page!
 			</p>
 			<ul>
+				{React.Children.toArray(
+					data.map(({ name }) => <ListItem name={name} />),
```

Comment:
Also, it is a good practice to define a [key](https://reactjs.org/docs/lists-and-keys.html#keys) for each item when you have a list of items :slightly_smiling_face:

Reply:
We chose the `React.Children.toArray` method to automatically allow React to assign the keys when rendering a list of children. But it is OK, as @Mama-simba and I can reverse it to the method we all know. 

@pmbanugo please check this.

Reply:
Oh, cool, I didn't know that :smiley: